products/casp/main.pm

```diff
@@ -103,19 +108,27 @@ sub load_feature_tests {
 }
 
 # ==== Installation workflow ====
-load_boot_tests;
+if (get_var('AUTOYAST')) {
+    load_boot_tests;
+    load_autoyast_tests;
+    loadtest 'casp/first_boot';
+    load_feature_tests;
+}
+else {
+    load_boot_tests;
 
-if (check_var('FLAVOR', 'DVD')) {
-    if (get_var('EXTRA', '') =~ /RCSHELL/) {
-        load_rcshell_tests;
-        return 1;
+    if (check_var('FLAVOR', 'DVD')) {
+        if (get_var('EXTRA', '') =~ /RCSHELL/) {
+            load_rcshell_tests;
+            return 1;
+        }
+        load_inst_tests;
```

Comment:
Please keep main.pm simple by reusing existing structure. All you need to change is line 125 to:
if (get_var('AUTOYAST')) {
  loadtest 'autoyast/installation';
} else {
  load_inst_tests;
}